sematic/examples/add/__main__.py

```diff
@@ -1,15 +1,28 @@
 # Standard Library
+import argparse
 import logging
 
 # Sematic
+from sematic import CloudResolver, LocalResolver
 from sematic.examples.add.pipeline import pipeline
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser("Sematic add example")
+    parser.add_argument("--cloud", type=bool, action="store_true", default=False)
+    parser.add_argument("--attach", type=bool, action="store_true", default=False)
```

Comment:
The argument name should be in line with existing naming conventions. `detach` already exists, so we should use that instead.

Reply:
The issue is that `detach=True` is the default, so we can't have a `--detach` option that set it to... `False`?

Reply:
Could you please clarify?

Reply:
`detach=True` is already the default. So what would a `--detach` option do?

Reply:
If you're concerned about flipping the default behavior of the flag / resolver, this is already done in other pipelines: [example](https://github.com/sematic-ai/sematic/blob/main/sematic/examples/bazel/bazel.py#L15-L21).

Reply:
Done.